### ISSUE TITLE
Build failing after upgrading to `core_extended v0.12.0`

### ISSUE THREAD

**Issue by SaswatPadhi** [janestreet/core_extended#22]
Hello.

I initially reported this issue (ocaml/dune#2010) to the `dune` developers, but may be this is actually an issue with the `core_extended` library?

With `v0.11.0` of `core_extended`, I see the following:
```
$ ocamlfind list | grep core_extended
core_extended       (version: v0.11.0)
core_extended.base64 (version: v0.11.0)
core_extended.selection (version: v0.11.0)

$ dune installed-libraries | grep core_extended
core_extended                       (version: v0.11.0)
core_extended.base64                (version: v0.11.0)
core_extended.selection             (version: v0.11.0)
```

But with `v0.12.0`, I see:
```
$ ocamlfind list | grep core_extended
core_extended       (version: n/a)
core_extended.base64 (version: v0.12.0)
core_extended.delimited_kernel (version: v0.12.0)
core_extended.find_files (version: v0.12.0)
core_extended.selection (version: v0.12.0)

$ dune installed-libraries | grep core_extended
core_extended.base64                    (version: v0.12.0)
core_extended.delimited_kernel          (version: v0.12.0)
core_extended.find_files                (version: v0.12.0)
core_extended.selection                 (version: v0.12.0)
```

Since the `core_extended` library is not found (but `core_extended.XXX` are!), my application that depends on this library fails to build now.

My `opam config report` is:
```
# opam config report
# opam-version      2.0.3 
# self-upgrade      no
# system            arch=x86_64 os=linux os-distribution=ubuntu os-version=18.04
# solver            builtin-mccs+glpk
# install-criteria  -removed,-count[version-lag,request],-count[version-lag,changed],-changed
# upgrade-criteria  -removed,-count[version-lag,solution],-new
# jobs              3
# repositories      1 (http) (default repo at a4d2989b)
# pinned            0
# current-switch    4.07.1+flambda
```

**Comment by SaswatPadhi** [comment]
Bump.

I was able to resolve most of the issues, but I cannot seem to find where `Bitarray` is now located. The [announcement here](https://discuss.ocaml.org/t/ann-v0-12-release-of-jane-street-packages/3499) doesn't list that either.

**Comment by snowleopard** [comment]
@SaswatPadhi Sorry it took us so long to reply. Indeed, it looks like `bitarray` isn't currently released on OPAM at all. 

Just to double-check: is this still an issue for you?

**Comment by SaswatPadhi** [comment]
Hi @snowleopard.

No worries. I am using the [`bitv`](https://github.com/backtracking/bitv) package now, so please feel free to close this issue.

**Comment by snowleopard** [comment]
Thanks, @SaswatPadhi! I'll keep this issue open for now. I think we should release `bitarray` on OPAM, I guess, under the name `core_extended.bitarray` for consistency with the rest of this repository.

**Comment by aalekseyev** [comment]
The fix was pushed in 907356c61785f53cdaa68445eb22a2231fef6774.
